### ISSUE TITLE
Retrieve information from backend for overview-page of digital identities

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#30]
## User story
1. As a {}
2. I want / need {}
3. So that {}

## Acceptance criteria
* Criterion 1
* Criterion 2
* ...

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by veitmo** [comment]
deleted because of duplicate
